native/packages/react-native-bpk-component-icon/BpkIcon.podspec

```diff
@@ -8,6 +8,7 @@ Pod::Spec.new do |s|
   s.homepage       = "https://github.com/Skyscanner/backpack/tree/master/packages/react-native-bpk-component-icon"
   s.license        = "Apache License 2.0"
   s.author         = { "Backpack" => "backpack@skyscanner.net" }
+  s.source         = { :path => './' }
   s.resources      = "./node_modules/bpk-svgs/dist/font/*.ttf"
```

Comment:
Although now i am looking at this:
```
s.resources      = "./node_modules/bpk-svgs/dist/font/*.ttf"
```

And wondering if it's correct 🤔  

I guess it works for you?

Reply:
yes, it's installing everything properly without complaining 🤞

Reply:
Yeah this seems problematic since it relies on the resolution/deduping by NPM

Reply:
```
node_modules/
  react-native-bpk-component-icon/
    BpkIcon.podspec
  bpk-svgs/
```
^ thats how the dir structure should look after `npm install`

but that `BpkIcon.podspec` points to a relative path (`./node_modules/bpk-svgs/dist/font/*.ttf`):
 ```
node_modules/
  react-native-bpk-component-icon/
    node_modules/
      bpk-svgs/                  // < ---- almost like it's looking here
    BpkIcon.podspec
  bpk-svgs/
```